vm-mock/vm.js

```diff
@@ -265,6 +283,12 @@ export default function createMockedABI(memory, createImports, instantiate, bina
         return true;
       },
 
+      assembly_script_hash_sha256(aPtr) {
+        const stringToHash = byteArrToUTF8String(getArrayBuffer(aPtr));
```

Comment:
stringToHash is not well named !

Reply:
Hi Pierre, thanks for the review. Is decodedString better ?